ext/flexmailer/tests/phpunit/Civi/FlexMailer/ClickTrackerTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\HookInterface;
+use Civi\Test\TransactionalInterface;
+
+use Civi\FlexMailer\ClickTracker\TextClickTracker;
+use Civi\FlexMailer\ClickTracker\HtmlClickTracker;
+use Civi\FlexMailer\ClickTracker\BaseClickTracker;
+
+/**
+ * Tests that URLs are converted to tracked ones if at all possible.
+ *
+ * @group headless
+ */
+class Civi_FlexMailer_ClickTrackerTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
```

Comment:
I think this is the test fail  \PHPUnit_Framework_TestCase is legacy - from the interweb

"you will need to refactor things like \PHPUnit_Framework_TestCase to \PHPUnit\Framework\TestCase"

Note if civix created this for you you  might have an old version of civix